src/bluesearch/entrypoint/database/parse.py

```diff
@@ -40,9 +46,12 @@ def run(
     Parameter description and potential defaults are documented inside of the
     `get_parser` function.
     """
+    parser_inst: ArticleParser
```

Comment:
Shouldn't `mypy` infer this automatically?

Reply:
I totally agree that it should, but unfortunately it doesn't. Without it, `mypy` thinks that the same variable assigned tow lines above may be re-assigned now to a different type, even if there's a clear `elif` in the middle. 

So I have to leave this type definition here or `mypy` will rant as follows:
```python
src/bluesearch/entrypoint/database/parse.py:48: 
error: Incompatible types in assignment (expression has type "PubmedXMLParser", variable has type "CORD19ArticleParser")  [assignment]
```